app/javascript/components/subject/Jurisdiction.js

```diff
@@ -0,0 +1,202 @@
+        <Modal.Body>
+          <p>
+            Are you sure you want to change jurisdiction from &quot;{this.props.jurisdictionPaths[this.state.original_jurisdiction_id]}&quot; to &quot;
+            {this.state.jurisdiction_path}&quot;?
+            {this.state.assigned_user !== '' && <b> Please also consider removing or updating the assigned user if it is no longer applicable.</b>}
+          </p>
+          {this.props.has_dependents && (
+            <React.Fragment>
+              <p className="mb-2">Please select the records that you would like to apply this change to:</p>
+              <Form.Group className="px-4">
+                <Form.Check
+                  type="radio"
+                  className="mb-1"
+                  name="apply_to_group"
+                  id="apply_to_group_no"
+                  label="This monitoree only"
+                  onChange={this.handleChange}
+                  checked={!this.state.apply_to_group}
+                />
+                <Form.Check
+                  type="radio"
+                  className="mb-3"
+                  name="apply_to_group"
+                  id="apply_to_group_yes"
+                  label="This monitoree and all household members"
+                  onChange={this.handleChange}
+                  checked={this.state.apply_to_group}
+                />
+              </Form.Group>
+            </React.Fragment>
+          )}
+          <Form.Group>
+            <Form.Label>Please include any additional details:</Form.Label>
+            <Form.Control as="textarea" rows="2" id="reasoning" onChange={this.handleChange} />
+          </Form.Group>
+        </Modal.Body>
+        <Modal.Footer>
+          <Button variant="secondary btn-square" onClick={toggle}>
+            Cancel
+          </Button>
+          <Button variant="primary btn-square" onClick={submit} disabled={this.state.loading}>
+            {this.state.loading && (
+              <React.Fragment>
+                <span className="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>&nbsp;
+              </React.Fragment>
+            )}
+            Submit
+          </Button>
+        </Modal.Footer>
+      </Modal>
+    );
+  }
+
+  render() {
+    return (
+      <React.Fragment>
+        <div className="disabled">
+          <Form.Label className="nav-input-label">
+            ASSIGNED JURISDICTION
+            <InfoTooltip tooltipTextKey="assignedJurisdiction" location="right"></InfoTooltip>
+          </Form.Label>
+          <Form.Group className="d-flex mb-0">
+            <Form.Control
+              as="input"
+              id="jurisdiction_id"
+              list="jurisdictionPaths"
+              autoComplete="off"
+              className="form-control-lg"
+              onChange={this.handleChange}
+              onKeyPress={this.handleKeyPress}
+              value={this.state.jurisdiction_path}
+            />
+            <datalist id="jurisdictionPaths">
```

Comment:
Is it time to stop using `datalists` here and convert to the React dropdowns you use elsewhere?

Reply:
I'm going to say no on this one mainly because there is a LOT of manipulating that needs to happen to get those to work.  They're great if you need advanced functionality for the dropdown (typing, multi select, etc) but these are so basic it would likely be more trouble than it is worth to do

Reply:
That's fine with me - we can consider that out of scope of this task then. Would still be nice to do eventually I think though, since datalists aren't great looking. I can put something on the backlog.